site/docs/v1/tech/installation-guide/docker.adoc

```diff
@@ -217,6 +203,7 @@ Here are the steps to do so:
 * In the `volumes:` section of the `fusionauth` service, add `- ./kickstart:/usr/local/fusionauth/kickstart`.
 * Modify `.env` and add the Kickstart configuration variable: `FUSIONAUTH_APP_KICKSTART_FILE=/usr/local/fusionauth/kickstart/kickstart.json`. This path should be what the Docker container expects, not the path on the host.
 * Configure `docker-compose.yml` to pass the environment variable set by `.env` to the container. Do this by adding `FUSIONAUTH_APP_KICKSTART_FILE: ${FUSIONAUTH_APP_KICKSTART_FILE}` to the `environment` section of the `fusionauth` service. 
+* `docker-compose up`
 
 The following is an example `docker-compose.yml` file configuring FusionAuth to run the commands in a `kickstart.json` at startup.
```

Comment:
Should we break this up into a separate includes file?  Or add this to the public container repo and just pull in the raw github files as we did with the main compose examples?

Reply:
We could use a separate include file, but what I really want is the ability to inject things into the current standard docker compose file (so that any changes outside of the kickstart env var/dir location are propagated) and I don't think asciidoctor can do that.